src/in_memory_time_series_storage.jl

```diff
@@ -21,7 +21,7 @@ end
 
 function InMemoryTimeSeriesStorage()
     storage = InMemoryTimeSeriesStorage(Dict{UUIDs.UUID, _TimeSeriesRecord}())
-    @info "Created InMemoryTimeSeriesStorage"
+    @debug "Created InMemoryTimeSeriesStorage"
```

Comment:
Should we delete this completely? Seems that it can create a huge amount of entries into the logs.

Reply:
It is only generated when the System is created.